editor/post-title/index.js

```diff
@@ -76,8 +85,19 @@ class PostTitle extends Component {
 		this.setState( { isSelected: false } );
 	}
 
-	handleClickOutside() {
-		this.setState( { isSelected: false } );
+	setFocused( focused ) {
+		this.setState( { hasFocusWithin: focused } );
+		if ( focused ) {
+			this.props.clearSelectedBlock();
+		}
+	}
+
+	handleFocusOutside( ) {
```

Comment:
Styling note: There should be no spaces within the parentheses:

>No filler spaces in empty constructs (e.g., `{}`, `[]`, `fn()`).

https://make.wordpress.org/core/handbook/best-practices/coding-standards/javascript/#spacing

I'll plan to look and see if we can enforce this by ESLint.

Reply:
Noted.

Reply:
Fixed.